src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+
+#include "Subscription.h"
+
+#include <string>
+#include <sstream>
+#include <vector>
+
+#include "common/JsonHelper.h"
+
+using std::string;
+using std::vector;
```

Comment:
Iustead of 'using std::xxx", I would propose to just maintain 'std::' where necessary.
ALL the code of the broker does that right now and I see no reason to change that behavior.
Especially not for a different behavior that contains less information